docs/program/apis/_index.md

```diff
@@ -367,6 +367,8 @@ Method Name | Description
 [`Move`](/services/motion/#move) | Move multiple components in a coordinated way to achieve a desired motion.
 [`MoveSingleComponent`](/services/motion/#movesinglecomponent) | Move a single component "manually."
 [`GetPose`](/services/motion/#getpose) | Get the current location and orientation of a component.
+[`MoveOnMap`](/services/motion/#moveonmap) | Move a component to a `Pose` in respect to the origin of a [SLAM](/services/slam/) map.
+[`MoveOnGlobe`](/services/motion/#moveonglobe) | Move a component to a destination GPS point. Use a [Movement Sensor](/components/movement-sensor/) to measure the robot's GPS coordinates.
```

Comment:
@npentrel I was originally thinking `Move a component to a specific latitude and longitude, using a [Movement Sensor](/components/movement-sensor/) to determine the location.` for this line but i think that could be a bit overwhelming. Thought this was easier to parse, suggestions welcome

Reply:
sounds good to me.